addon/components/frost-select.js

```diff
@@ -119,7 +120,7 @@ export default Component.extend({
       renderTarget: 'frost-select',
       role: 'button',
       tabIndex: 0,
-
+      debounceInterval: 0,
```

Comment:
@sglanzer if we don't make the default 0 don't we risk breaking consumers who could theoretically rely on every keystroke? I know some of my use cases rely on client-side filtering and it is nice having the instant feedback loop as the user types.